README

```diff
@@ -15,13 +15,21 @@ Permission is granted hereby, to copy, share, and modify. Use as is fit,
 free or for profit. These rights, none can deny!
 
 
+Change Log:
+====================
 
-(c) 2005 Alexander E Genaud
+June 14, 2011:
+ - Claiming "We’ve done our best" might have legal implications. Best to remove and replace with a statement stopping all claims. (Xeoncross)
+
+
+
+-------------------------------------
+
+(c) 2011 Company Name
 
 This work ‘as-is’ we provide.
 No warranty express or implied.
-We’ve done our best,
-to debug and test.
+Therefore, no claim on us will abide.
```

Comment:
Just came across this PR, and although this is moot by now, I thought I'd point out that this line would have a better metric as "Thus no claim on us will abide" (which still doesn't make the verse a limerick, but hey).